src/api-service/__app__/onefuzzlib/config.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from typing import Optional, Tuple
+
+from onefuzztypes.events import EventInstanceConfigUpdated
+from onefuzztypes.models import InstanceConfig as BASE_CONFIG
+from pydantic import Field
+
+from .azure.creds import get_instance_name
+from .events import send_event
+from .orm import ORMMixin
+
+
+class InstanceConfig(BASE_CONFIG, ORMMixin):
```

Comment:
I think the PR is missing the initilization of the admins in the deployment script. Unless i missed that logic somewhere else

Reply:
You didn't miss it.  It's listed in the PR's todo list.

Reply:
added